cli/main.go

```diff
@@ -104,12 +104,22 @@ Example:
 
 func buildDefaultConfigPath() string {
 	var result string
+	var homeDirFile string
+	configFileName := "revive.toml"
+
+	configDirFile := filepath.Join(os.Getenv("XDG_CONFIG_HOME"), configFileName)
 	if homeDir, err := homedir.Dir(); err == nil {
-		result = filepath.Join(homeDir, "revive.toml")
-		if _, err := os.Stat(result); err != nil {
-			result = ""
-		}
+		homeDirFile = filepath.Join(homeDir, configFileName)
+	}
+
+	if _, err := os.Stat(configDirFile); err == nil {
+		result = configDirFile
+	} else if _, err := os.Stat(homeDirFile); err == nil {
+		result = homeDirFile
+	} else {
+		result = ""
 	}
+    print(result)
```

Comment:
This here seems unnecessary.

Reply:
sorry my bad, left that there by mistake.